localauth0.toml

```diff
@@ -9,7 +10,6 @@ birthdate = "2022-02-11"
 email = "developers@prima.it"
 picture = "https://github.com/primait/localauth0/blob/6f71c9318250219a9d03fb72afe4308b8824aef7/web/assets/static/media/localauth0.png"
 custom_fields = [
-    { name = "address", value = { String = "github street" } },
```

Comment:
I removed this custom field because address is in the id token specification and my `openidconnect` client complained about its structure.